src/pages/DisplayCards.tsx

```diff
@@ -1,23 +1,75 @@
 import React, { useState } from "react";
+import ReactTooltip from "react-tooltip";
 
 import { Card, Button, Typography, Layout } from "antd";
 
 import logo from "../assets/images/logo.svg";
+import edit from "../assets/images/edit.svg";
 import "../styles/DisplayCards.css";
 
-const initialValues = {
-  text: "Lorem ipsm dolor sit amet, consectetuer adipiscing elit, sed diam",
+export enum CardSide {
+  Left,
+  Right,
+}
+
+type Card = {
+  text: string;
+  isEditing: boolean;
+};
+
+const initialLeftCard: Card = {
+  text:
+    "(Left Card) Lorem ipsm dolor sit amet, consectetuer adipiscing elit, sed diam",
+  isEditing: false,
+};
+
+const initialRightCard: Card = {
+  text:
+    "(Right Card) Lorem ipsm dolor sit amet, consectetuer adipiscing elit, sed diam",
+  isEditing: false,
 };
 
 const DisplayCards: React.FC = () => {
-  const [state] = useState(initialValues);
+  const [leftState, setLeftState] = useState(initialLeftCard);
+  const [rightState, setRightState] = useState(initialRightCard);
   const onBackClick = () => {
     // TODO: Write code here to redirect to course info screen
   };
 
   const onSkipClick = () => {
     // TODO: Write code here to redirect to instructions screen
   };
+
+  const onEditClick = (side: CardSide) => {
+    var textElement;
```

Comment:
You probably want to use 'let' here instead of 'var'. I think that's the general rule that is followed.